build/app_template_files/createApp.sh.base

```diff
@@ -76,14 +71,13 @@ function usage()
   local appName=`basename $0`
   echoColor $TERM_COLOR_CYAN "Usage:"
   echoColor $TERM_COLOR_MAGENTA "$appName"
-  echoColor $TERM_COLOR_MAGENTA "   -t <Application Type> (native, hybrid_remote, hybrid_local)"
+  echoColor $TERM_COLOR_MAGENTA "   -t <Application Type> (native)"
```

Comment:
Keeping the type option even though there is only one right now, because we will soon introduce support for swift native app.
Or maybe the language (swift or objective-c) should be a different option?